hooks/checkout

```diff
@@ -16,15 +16,12 @@ echo "Setup completed successfully."
 echo "--- :key: Setting up git and ssh"
 if ! ssh-keygen -F github.com > /dev/null; then
   echo "github.com not found in known_hosts"
-  mkdir -p /tmp
-  TMP_FILE="$(mktemp /tmp/github_rsa.pub.XXXXXX)"
-  ssh-keyscan -t rsa github.com > "$TMP_FILE"
```

Comment:
Avoiding calls to `ssh-keyscan` to save startup time.

API calls will always be slower than hardcoding the public keys - but if automation should be used, then see `Github Rest API Meta` instead: https://docs.github.com/en/rest/meta/meta?apiVersion=2022-11-28#get-github-meta-information